### PR TITLE
Add multiple cost labels

### PR DESCRIPTION
**Pull request by aanil**

### CODE REVIEW

File: run_dir/static/js/pricing_quote.js [hunk 1]
```diff
@@ -95,18 +112,19 @@ app.component('v-pricing-quote', {
         },
         generate_quote:  function (event) {
           product_list = {}
+          var no_prod_error = 'Please add Products to generate a quote!';
           if(Object.keys(this.$root.quote_prod_ids).length > 0){
+            if(this.$root.any_errors)
+              this.$root.error_messages.pop()
```
Comment (alneberg) [inline run_dir/static/js/pricing_quote.js:118]: Didn't know you could skip the curly brackets... not sure I'm convinced it's a good thing for readability though?

File: status_app.py [hunk 2]
```diff
@@ -250,9 +250,7 @@ def __init__(self, settings):
             self.gs_users_db = couch["gs_users"]
             self.instruments_db= couch["instruments"]
             self.instrument_logs_db = couch["instrument_logs"]
-            self.pricing_components_db = couch["pricing_components"]
             self.pricing_exchange_rates_db = couch["pricing_exchange_rates"]
-            self.pricing_products_db = couch["pricing_products"]
```
Comment (alneberg) [inline status_app.py:255]: Thanks for this, and sorry for causing the error

File: run_dir/static/js/pricing_quote.js (hunk 1 above)
Comment (aanil) [reply run_dir/static/js/pricing_quote.js:118]: You can if its just a single line. I can add it if you want

File: run_dir/static/js/pricing_main.js [hunk 3]
```diff
@@ -39,8 +39,7 @@ const vPricingMain = {
 
             // Quote data
             quote_prod_ids: {},
-            quote_special_addition_value: 0,
-            quote_special_addition_label: '',
+            quote_special_addition_labels: {},
```
Comment (alneberg) [inline run_dir/static/js/pricing_main.js:42]: Maybe `quote_special_additions`?

File: run_dir/static/js/pricing_quote.js [hunk 4]
```diff
@@ -17,7 +19,12 @@ app.component('v-pricing-quote', {
                     Object.keys(this.$root.quote_prod_ids).length)
         },
         any_special_addition() {
-            return this.$root.quote_special_addition_label !== ''
+            for ([index, label] of Object.entries(this.$root.quote_special_addition_labels)){
```
Comment (alneberg) [inline run_dir/static/js/pricing_quote.js:22]: `quote_special_additions` here as well

File: run_dir/static/js/pricing_quote.js [hunk 5]
```diff
@@ -83,9 +94,15 @@ app.component('v-pricing-quote', {
             this.$root.quote_special_percentage_label = ''
             this.$root.quote_special_percentage_value = 0
         },
-        reset_special_addition() {
-            this.$root.quote_special_addition_label = ''
-            this.$root.quote_special_addition_value = 0
+        add_cost_label: function(){
+          this.$root.quote_special_addition_labels[this.cLabel_index] = { name: '', value: 0 };
```
Comment (alneberg) [inline run_dir/static/js/pricing_quote.js:98]: `quote_special_additions`

File: run_dir/static/js/pricing_quote.js [hunk 6]
```diff
@@ -83,9 +94,15 @@ app.component('v-pricing-quote', {
...
+          this.cLabel_index++;
+        },
+        remove_cost_label: function(index){
+          delete this.$root.quote_special_addition_labels[index];
```
Comment (alneberg) [inline run_dir/static/js/pricing_quote.js:102]: `quote_special_additions`

**Review by alneberg — APPROVED** [review]
Great, thanks!

**Pull request merged**